pkg/operator/ceph/pool/controller.go

```diff
@@ -542,3 +554,27 @@ func (r *ReconcileCephBlockPool) disableMirroring(pool string) error {
 
 	return nil
 }
+
+func (r *ReconcileCephBlockPool) isAnyRadosNamespaceMirrored(poolName string) (bool, error) {
+	logger.Debugf("list rados namespace in pool %q running in ceph cluster namespace %q", poolName, r.clusterInfo.Namespace)
+
+	list, err := cephclient.ListRadosNamespacesInPool(r.context, r.clusterInfo, poolName)
+	if err != nil {
+		return false, errors.Wrapf(err, "failed to list rados namespace in pool %q", poolName)
+	}
+	logger.Infof("rados namespace list %v", list)
+
+	for _, namespace := range list {
+		poolAndRadosNamespaceName := fmt.Sprintf("%s/%s", poolName, namespace)
```

Comment:
we might need to revisit this because for the case where namespace is empty but it mirror enabled to remote namespace (not for now)